doc/source/cookbook/opengl_ipython.py

```diff
@@ -1,30 +0,0 @@
-import yt
```

Comment:
This file is referenced explicitly in doc/source/cookbook/complex_plots.rst
I'm not against its removal, but the reference need be cleaned up too :)